Makefile

```diff
@@ -32,6 +35,9 @@ test: testdeps
 testrace: testdeps
 	go test -v -race -cpu 1,4 google.golang.org/grpc/...
 
+benchmark: benchdeps
+	go test benchmark/benchmark17_test.go benchmark/benchmark.go -benchmem -bench=. -timeout 1h | tee results && benchstat results && rm results
```

Comment:
Use `google.golang.org/grpc/benchmark/...` instead of `benchmark/benchmark17_test.go benchmark/benchmark.go`?